Update running-pinot-in-docker.md to include minion config

vipulchandraker opened the pull request.

vipulchandraker committed:
Update running-pinot-in-docker.md to include minion config

NihalJain commented on basics/getting-started/running-pinot-in-docker.md:
```diff
@@ -319,6 +319,16 @@ services:
 
+  pinot-minion:
+    image: ${PINOT_IMAGE:-apachepinot/pinot:1.2.0}
+    command: "StartMinion -zkAddress zookeeper-houseprices:2181"
```
1. Do you mean pinot-zookeeper:2181? Is this docker compose file tested? 
2. Could we add ports?
3. Can you also update relevant services which start up i.e. `docker container ls` output?
4. Also consider adding how to start minion via docker run in the section above docker compose section?

vipulchandraker replied:
@NihalJain I have made all the suggested changes and tested the docker compose file. Please review

vipulchandraker committed:
Update running-pinot-in-docker.md

1. updated the correct zookeeper address in Minion in docker compose.
2. Added docker run command for minion
3. Added minion container in docker container ls output
4. added minion ports in docker-compose file

xiangfu0 reviewed (approved):
LGTM - good addition of Pinot Minion configuration to Docker docs.
